Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -578,6 +578,51 @@ metroTicketBannerConfig state =
     config'
 
 
+contactSupportPopUpConfig :: ST.HomeScreenState -> PopUpModal.Config
+contactSupportPopUpConfig state =
```

Comment:
put this component inside commonComponentConfig. As it is being declared in TripDetailsScreen also.